### DISCUSSION TITLE
Live Preview

### DISCUSSION THREAD

**Discussion by arachnidus** [brackets-cont/brackets#273] [Q&A]
Since release 2.2 I have two issues with Live Preview on Windows

- I have to start Brackets 2.2 in administrator mode to write/create a datadirectory
- My user name contain spaces and Chrome give errors by opening datadirectory. This issue is not available when I start Chrome by shortcut.

**Comment by abose** [comment]
The issue with spaces have been addressed with latest version of brackets. Please update and see if that fixes the issue.

**Comment by arachnidus** [comment]
I am using Release 2.2 build 2.2.0-17942 (master c7a435e49) build timestamp: Sat Mar 11 2023 13:04:22 GMT +0530. When I use "check for updates" I get the message "you are running the latest version of brackets. Where Can I download the most latest build?

**Reply by abose** [reply]
`2.2.1` is the latest release, https://github.com/brackets-cont/brackets/releases/tag/v2.2.1
It was rolled out specifically to fix the space issue. Please download it from the link.
